hello 

#  unsolved problem
#  when run cross_test.py, the size of used internal memory is more and more big, and finally momery is not supported.
#  it has solved.


